ansible/roles/devel/tasks/main.yaml

```diff
@@ -17,3 +17,30 @@
   ansible.builtin.command: git lfs install
   when: "'filter.lfs.required' not in git_lfs__git_global_config.config_values"
   changed_when: true
+
+- name: Install plotjuggler
+  become: true
+  ansible.builtin.apt:
+    name:
+      - ros-{{ rosdistro }}-plotjuggler-ros
+    state: latest
+    update_cache: true
```

Comment:
Are there any reasons why you removed the role for plotjuggler, pre-commit, clang-format, and golang and git_lfs?

These packages are optional packages so I thought it would be useful to keep them as different roles so that users can configure the combination to install at the playbook level (e.g., when they create a playbook for their own project).

Reply:
I think there is a trade-off between having all non-essential packages as separate roles and keeping them as one optional role. 

IMHO, if we keep them as separate roles it gives the flexibility but it also may bloat tracking of all non-essential packages in the future. Also, by default calling `setup-dev-env.sh -y `installs all of these packages anyways .